### ISSUE TITLE
DEFINITION  update and REMOVE PARENT  retrograde protein transport, ER to cytosol (GO:0030970)  

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#21269]
retrograde protein transport, ER to cytosol (GO:0030970) 
is defined
The directed movement of unfolded or misfolded proteins from  the endoplasmic reticulum to the cytosol through the translocon.            [PMID:11994744]

but is not a descendant of

ubiquitin-dependent ERAD pathway (GO:0030433) 
The series of steps necessary to target endoplasmic reticulum  (ER)-resident proteins for degradation by the cytoplasmic proteasome.  Begins with recognition of the ER-resident protein, includes  retrotranslocation (dislocation) of the protein from the ER to the  cytosol, protein ubiquitination necessary for correct substrate  transfer, transport of the protein to the proteasome, and ends with  degradation of the protein by the cytoplasmic proteasome.            [PMID:14607247, PMID:19520858]

it seems that it should be?

**Comment by pgaudet** [comment]
GO:0030970 sounds more general than GO:0030433, but maybe there is a mistake in the definiton of GO:0030970 - the transport is via the retrotranslocon, not the translocon (see also https://en.wikipedia.org/wiki/Translocon)

But in this case - are those 2 terms really different ?

**Comment by ValWood** [comment]
Probably the same, see:
https://github.com/geneontology/go-ontology/issues/20914

**Comment by ValWood** [comment]
There are papers that suggest the involvement of the translocon
https://europepmc.org/article/MED/10635318 (Old)
but newer papers do not mention the translocon
https://europepmc.org/article/MED/24807418
 
Suggest
i) revise def to 
The retrotranslocation of unfolded, misfolded or damaged proteins from the endoplasmic reticulum to the cytosol.

ii) remove parent "protein exit from the ER". The ER exit site is the site where new COPII? vesicles form, and this exit site/process  is completely different (related to vesicle-mediated transport, not  ERD)

ii) The ERAD paernt is OK< as this will be the one retained after the merges

**Comment by pgaudet** [comment]
Hi @ValWood 
I am not sure retrograde transport always leads to degradation, see 

- https://pubmed.ncbi.nlm.nih.gov/36933283: movement of incorrectly localized proteins & recycling
- https://pubmed.ncbi.nlm.nih.gov/36895788: recycling of proteins
- https://pubmed.ncbi.nlm.nih.gov/34012437: silencing STING signaling pathway

So, I dont think this parent would be right. 


That being said, when I look for 'retrograde transport', it seems like this refers to 'Golgi-to-ER', not ER to cytosol. I seems like 'retrotranslocation' and 'retrograde transport' mean different things? ie GO:0030970 term label is wrong WRT its definition?

**Comment by ValWood** [comment]
Yes I think something got mixed up over time. The term more costly matches the exact synonym

protein retrotranslocation, ER to cytosol | exact

I don't think this is usually called retrograde transport. That seems to be 
https://www.ebi.ac.uk/QuickGO/term/GO:0006890
retrograde vesicle-mediated transport, Golgi to endoplasmic reticulum
(i.e vesicle mediated)

This term seems to be a slight conflation, I'm not sure that it is necessary, maybe obsoletion is better recommending either

ERAD (ER to cytosol for degradation) the ubiquitin mediated extraction, and degradation

or 
retrograde vesicle-mediated transport, Golgi to endoplasmic reticulum
for vesicle mediate d recycling.


The term was definitely intended for the ERAD pathway translocation

**Comment by pgaudet** [comment]
> The term was definitely intended for the ERAD pathway translocation

But the reference cited for the term (in the def xref) talks about retrotranslocation, not retrograde transport... 

We need to look at annotations to know how people used these terms- there are 'only' 60 annotations, see https://github.com/geneontology/go-annotation/issues/4826

**Comment by ValWood** [comment]
Yes, I think the term was *intended* for retrotranslocation but he use of retrograde in the name might have confused the usage. If all of the usage conforms to ERAD, it's fine to swap the names

**Comment by pgaudet** [comment]
and make sure references are appropriate

**Comment by ValWood** [comment]
CLosing, will be addressed by later tickets
